src/caffe/layers/python_layer.cpp

```diff
@@ -0,0 +1,56 @@
+#ifdef WITH_PYTHON_LAYER
+
+#include "caffe/python_layer.hpp"
+
+#include <boost/thread.hpp>
+#include <vector>
+
+namespace caffe {
+
+class aquire_py_GIL {
```

Comment:
Yeah, I think any classes we add should follow our CamelCase style.  (Also, should be spelled `acquire`.)